packages/run-protocol/src/centralSupply.js

```diff
@@ -31,7 +31,10 @@ export const start = async (zcf, { feeMintAccess }) => {
 
   return {
     creatorFacet: Far('creator', {
-      getBootstrapPayment: () => bootstrapPayment,
+      getBootstrapPayment: () => {
+        zcf.shutdown('payment retrieved');
+        return bootstrapPayment;
```

Comment:
shutdown is synchronous and seems to kill the containing vat. Can the function return anything after that? Does this only work in tests?

It's also counter to our approach for contracts to return payments from API methods rather than as payouts. Why are we doing it this way?

Reply:
I'd be surprised if anyone is currently looking for those results. 

OTOH, someone certainly wants the bootstrapPayment. If that's accessible outside of tests, then it works.

Reply:
> Why are we doing it this way?

@dtribble reminds us: offer handling used to have fees etc. that complicate things, and the plan is that they will again in due course.